.github/workflows/pr_checks.yml

```diff
@@ -13,6 +13,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v1
         with:
-          go-version: 1.20
+          go-version: 1.21
```

Comment:
maybe you can also update the following ;P
- https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4
- https://github.com/actions/setup-go?tab=readme-ov-file#v4